.env

```
@@ -5,7 +5,7 @@ ALLOW_ANONYMOUS=1
 #CERT_FILE=fixtures/tls/server.crt
 #KEY_FILE=fixtures/tls/server.key
 CORS_ALLOWED_ORIGINS=http://localhost:3000,http://localhost:8000
-DB_PATH=
+HISTORY_DNS=bolt://update.db
```

```suggestion
HISTORY_DSN=bolt://update.db
```

What do you think about `STREAM_URL` (similar to Heroku/Symfony `DATABASE_URL`, or even `ADAPTER_URL` (adapter is the term used by socketio), because it's not only about history.  Or `TRANSPORT_URL`?

If we choose `adapter` instead of `stream`, we should also rename the interface.

---

You're right, I first kept `HISTORY_` for consistency with other parameter. But once we removed them, it's not necessary anymore.
I don't like `adapter` because it's too generic and could be anything.

The parameter should be understandable from a user POV, `STREAM_URL` may not talk to everybody. I'll note

IMHO The parameter don't have the same prefix as the interface (like `MAILER_DNS` for `Transport` interface in symfony)